procrastinate/app.py

```diff
@@ -128,6 +130,9 @@ def _register(self, task: "tasks.Task") -> None:
             )
             self.queues.add(task.queue)
 
+    def _register_builtin_tasks(self) -> None:
+        builtin_tasks._register_builtin_tasks(self)
```

Comment:
Le _ dans le nom de la méthode, c'est utile, mais celui dans la fonction... Je sais pas... Ca veut pas dire non, mais, je me demande.

Reply:
:gb:

Which one :> ? The `builtin_tasks._register_builtin_tasks` function or the `App._register_builtin_tasks` method?
For the method I think it makes sense. For the function I don't really have an opinion, whatever you prefer :)

Reply:
(hey, for the record, I translated to :gb: less than a minute after the original comment :open_mouth: )
I'm talking about the function, not the method.

I'm ok leaving the underscore because people might look at the module when searching for a builtin task name, but... I wouldn't oppose a commit removing it either :D

Reply:
I removed the leading `_`